src/main/java/sirius/db/jdbc/schema/SchemaTool.java

```diff
@@ -187,11 +187,14 @@ private void dropUnusedTables(List<SchemaUpdateAction> result,
                                   List<Table> targetSchema) {
         for (Table table : currentSchema) {
             if (findInList(targetSchema, table) == null) {
-                SchemaUpdateAction action = new SchemaUpdateAction(realm);
-                action.setReason(NLS.fmtr("SchemaTool.tableUnused").set(KEY_TABLE, table.getName()).format());
-                action.setDataLossPossible(true);
-                action.setSql(dialect.generateDropTable(table));
-                result.add(action);
+                String sql = dialect.generateDropTable(table);
```

Comment:
update @Nullable annotations and javadoc in DatabaseDialect to document this behaviour...